docs/concepts/packaging-types.adoc

```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Build hollow jars
+
+WildFly Swarm ships three pre-built hollow jar types, for different use cases:
+
+* full: The main Java EE related capabilities
+* web: A smaller functional scope, focused on web technologies
+* microprofile: The capabilities belonging the microprofile specification
```

Comment:
Maybe the phrase "Functionality defined by all Eclipse MicroProfile specifications"?

Reply:
I had a comment here that disappeared for some reason. Not sure if someone seen it. I'll try to repeat.

This one line has these issues: "belonging _to_ the _MicroProfile_ [or Eclipse MicroProfile] specitication_s_".

But in general, I think this list is too vague. I think we could at least write:

- `full`: all the Java EE capabilities
- `web`: limited set of the Java EE capabilities, focused on web-related technologies
- `microprofile`: same as `web` plus all the Eclipse MicroProfile capabilities

Reply:
i've gone with @kenfinnigan suggestion

Reply:
Ah and I was totally wrong, our MP hollow JAR _isn't_ a superset of `web`. Thanks :-)